### ISSUE TITLE
Transparent tabs

### ISSUE THREAD

**Issue by Shaltz** [bigbn/drop-down-terminal-x#68]
Hi, 

I moved from 18.04 to 20.04 (keeping the same /home) and for some reason, the tabs are now transparent... that makes them unreadable when the background is not totally uniform and dark

I did uninstall tDDTX, reset all the preferences using dconf editor and reinstalled it, but the problem remains :(

![Screenshot from 2020-05-05 10-31-47](https://user-images.githubusercontent.com/10232812/81048782-42dd8b80-8ebd-11ea-8523-5b9c7e2c72ac.png)

**Comment by nirmalhk7** [comment]
Facing the same issue. Excellent plugin otherwise.

**Comment by Svad** [comment]
Same for me. 
![2020-05-09_05-45](https://user-images.githubusercontent.com/458974/81462038-7455a000-91b8-11ea-948f-0f0dbac3c6bf.png)

**Comment by Shaltz** [comment]
@bigbn 
Hi, no idea what is causing this issue and how to fix it ?

**Comment by bigbn** [comment]
No. I need to check it by myself. This need a time to install ubuntu and check what there is going on

**Comment by Shaltz** [comment]
@bigbn 
is there anything we can give you to help you out ? some logs, something ?

**Comment by Shaltz** [comment]
no news about this issue ??

**Comment by bigbn** [comment]
I'm sorry, not yet

**Comment by Dmitriy0712** [comment]
Nice plugin! \
Have the same issue. Found a workaround, it's ugly, but helps :) \
Added this:
```
window>*>* {
    background-color: rgba(36, 31, 49, 0.94); 
}
```
to gtk.css (`~/.local/share/gnome-shell/extensions/drop-down-terminal-x@bigbn.pro/gtk.css` in my case)

**Comment by Shaltz** [comment]
I don't believe it's related to the 20.04 version... I had to reinstall a system on a brand new computer, 
I went to ubuntu 18.04.5 and the problem is there too... 

![Screenshot from 2021-02-03 17-18-59](https://user-images.githubusercontent.com/10232812/106777037-e8887b80-6644-11eb-96b3-4c2b3e433e13.png)
